newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalFileResourceMover.js

```diff
@@ -25,6 +30,43 @@ type Options = {|
   onProgress: (number, number) => void,
 |};
 
+const generateUnusedFilepath = (
```

Comment:
I got confused at first read what unused meant.
Should this be named more "generateAvailableFilepath"?
(or maybe add a comment to the function)

Reply:
The more I read it, the more I get used to it, so I guess it's fine